include/nuttx/video/fb.h

```diff
@@ -315,11 +319,15 @@ struct fb_videoinfo_s
 
 struct fb_planeinfo_s
 {
-  FAR void  *fbmem;       /* Start of frame buffer memory */
-  size_t     fblen;       /* Length of frame buffer memory in bytes */
-  fb_coord_t stride;      /* Length of a line in bytes */
-  uint8_t    display;     /* Display number */
-  uint8_t    bpp;         /* Bits per pixel */
+  FAR void  *fbmem;        /* Start of frame buffer memory */
+  size_t     fblen;        /* Length of frame buffer memory in bytes */
+  fb_coord_t stride;       /* Length of a line in bytes */
+  uint8_t    display;      /* Display number */
+  uint8_t    bpp;          /* Bits per pixel */
+  uint32_t   xres_virtual; /* Virtual Horizontal resolution in pixel columns */
+  uint32_t   yres_virtual; /* Virtual Vertical resolution in pixel rows */
+  uint32_t   xoffest;      /* X offset */
+  uint32_t   yoffset;      /* Y offset */
```

Comment:
Please, improve the documentation for these members to make them more intuitive. As a reference, see the documentation for the Linux struct:
https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/fb.h#L243
![image](https://user-images.githubusercontent.com/38959758/135271734-04ad51ae-4e5e-4953-939d-36d578c17a13.png)